src/components/ForgotPassword.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import "../styles/login.css"
```

Comment:
import style file in common parent instead of importing in each child components